api/product.rb

```diff
@@ -84,12 +84,15 @@ def product_full_name
       end
     end
 
+    # Default to most general version that exists for the product
+    # If GA is present, use that, else beta, else alpha
     def default_version
```

Comment:
I'll let @chrisst give a final opinion on this.

This kind of change seems small enough + self-contained enough that unit tests would be great for it, even though we're not good at unit testing.